wake/environment.wake

```diff
@@ -1,78 +1 @@
-#################################################################################
-## Logic for handling custom environment
-##
-## Do not edit below this line
-#################################################################################
```

Comment:
Is this similar to what was in environment-example-sifive?

Reply:
Yeah, this essentially got factored out _into_ environment-example-sifive in https://github.com/sifive/environment-example-sifive/pull/1 (both this and that PR were submitted in the middle of last November).